taskcat/project_config/tools.py

```diff
@@ -0,0 +1,201 @@
+
+def _add_qs_info(parameter):
+    """Method to to add tcparameter for QS items"""
+    return_text = ''
+    match parameter.name.lower():
+        case "qss3bucketname":
+            return_text = (f'    {parameter.name}: $[taskcat_autobucket]'
+                           f'\r\n')
+        case "qss3bucketregion":
+            return_text = (f'    {parameter.name}: $[taskcat_current_region]'
+                           f'\r\n')
+    return return_text
+
+
+def _add_help_info(parameter):
+    """Method to add help info to the parameter object"""
+    attributes = [
+        "Description",
+        "AllowedPattern",
+        "AllowedValues",
+        "ConstraintDescription",
+        "Default",
+        "MaxLength",
+        "MaxValue",
+        "MinLength",
+        "NoEcho",
+        "Type"
+    ]
+    helptext = ""
+    for attribute in attributes:
+        if hasattr(parameter, attribute):
+            attr = getattr(parameter, attribute)
+            helptext += (f'    # {attribute}: '
+                         f'{str(attr)}\r\n')
+        # Remove empty lines
+        lines = helptext.split('\n')
+        non_empty_lines = [line for line in lines if line.strip() != '']
+    return '\n'.join(non_empty_lines)
+
+
```

Comment:
This isn't something that's scalable; I recommend deferring to the CFN schemas and rmeove this entire function.

Reply:
I removed the verbose config option